external-crates/move/crates/move-compiler/src/parser/syntax.rs

```diff
@@ -1503,6 +1503,45 @@ fn parse_sequence_item(context: &mut Context) -> Result<SequenceItem, Box<Diagno
     ))
 }
 
+// Checks if parsing of a sequence should continue after encountering an error.
+fn continue_sequence_after_error(
```

Comment:
Nit: `should_continue_sequence_parsing` or similar?

Reply:
Changed to `should_continue_sequence_after_error`